app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -451,7 +468,8 @@ class MEJSPlayer {
       link_back_url: currentStreamInfo.link_back_url,
       qualityText: 'Stream Quality',
       defaultQuality: this.defaultQuality,
-      toggleCaptionsButtonWhenOnlyOne: true
+      toggleCaptionsButtonWhenOnlyOne: true,
+      startVolume: this.localStorage.getItem('startVolume') || 0.8
```

Comment:
@joncameron @davidschober Do you think we can default to volume at 100% instead of the mediaelement.js default of 80%?  I always find myself trying to crank it to 100% anyways.

Reply:
```suggestion
      startVolume: this.localStorage.getItem('startVolume') || 1.0
```

Reply:
@cjcolvar Yes, let's do that as the default.